.github/workflows/continuous_integration.yml

```diff
@@ -16,3 +16,9 @@ jobs:
     secrets:
       TEST_PYPI_API_TOKEN: ${{ secrets.TEST_PYPI_API_TOKEN }}
       PYPI_API_TOKEN: ${{ secrets.PYPI_API_TOKEN }}
+
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: psf/black@stable
```

Comment:
[question] my assumption is that this will cause a failure if any files aren't properly formatted. Is that correct?